deb.mk

```diff
@@ -61,7 +61,7 @@ deb-all: deb-buildstep deb-dokku deb-gems deb-pluginhook deb-sshcommand
 deb-setup:
 	echo "-> Updating deb repository and installing build requirements"
 	sudo apt-get update > /dev/null
-	sudo apt-get install -qq -y git ruby1.9.1-dev 2>&1 > /dev/null
+	sudo apt-get install -qq -y gcc git ruby1.9.1-dev 2>&1 > /dev/null
```

Comment:
Does gcc get installed only on the build host or will this also end up on a running web instance as well?